packages/sample-app-fabric/windows/SampleAppFabric/SampleAppFabric.cpp

```diff
@@ -39,6 +41,10 @@ _Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR
   RegisterAutolinkedNativeModulePackages(settings.PackageProviders());
   // Register any native modules defined within this app project
   settings.PackageProviders().Append(winrt::make<CompReactPackageProvider>());
+  // TODO: Can we make apps register this automatically?  (e.g. with autolinking)
+  // TODO: But ideally we don't load the M.RN.Xaml.dll at all if we're not using Xaml.
```

Comment:
Hmm, that would actually be an interesting thing to try to get autolinking to do.
Right now auto linking would end up loading the dll. -- Having said that, I wouldn't expect M.RN.Xaml.dll to be particularly big.  We will want to make sure that MUX doesnt' get loaded until you use a control though.